tutorials/shading/shading_reference/spatial_shader.rst

```diff
@@ -72,6 +72,8 @@ Render modes
 +---------------------------------+-----------------------------------------------------------------------+
 | **ambient_light_disabled**      | Disable contribution from ambient light and radiance map.             |
 +---------------------------------+-----------------------------------------------------------------------+
+| **shadow_to_opacity**           | Allows for overlaying shadows onto a camera feed (good for AR use).   |
```

Comment:
Im not sure this description properly captures what shadow_to_opacity does. It needs to be more descriptive about how this renders non-shadowed areas transparent with shadowed areas opaque.